lib/perl/Genome/Model/Tools/Somatic/FilterFalsePositives.pm

```diff
@@ -636,6 +665,10 @@ sub fails_homopolymer_check {
 sub readcount_program {
     my $self = shift;
     my $reference = $self->reference;
+    if($self->bam_readcount_version) {
+      my $bam_rc = "bam-readcount" . $self->bam_readcount_version;
```

Comment:
This is better than a hardcoded version! :smile:  A future improvement could be defer to `Genome::Model::Tools::Sam::Readcount` to get the path and/or run this.